libs/react/src/lib/modal/modal.stories.tsx

```diff
@@ -67,13 +67,34 @@ ModalDefault.args = {
   dismiss: 'Nope',
 }
 
-export const SlideOut: Story<ModalProps> = Template.bind({})
-SlideOut.args = {
+export const SlideOutSmall: Story<ModalProps> = Template.bind({})
```

Comment:
I don't know if this is the correct way of displaying different slide out versions?

Reply:
It's correct! However the whole documentation for modals could probably be improved.. But it's out of scope for this PR. Thanks for contributing <3

Reply:
Thank you for looking through the PR so fast!